src/app/components/user/hidUsers.controller.js

```diff
@@ -0,0 +1,135 @@
+hidControllers.controller('hidUsersCtrl', ['$scope', '$rootScope', '$location', '$routeParams', 'gettextCatalog', 'alertService', 'hrinfoService', 'userService', 'User', 'List',
```

Comment:
I'm not a fan of naming a user related controller hidUsers and putting it into a hidUsers.controller.js file... Either we keep this as a directive, in which case it should go in a user.directives.js file, or we make it a controller, in which case it should go in user.controllers.js (at least for now, controllers can be split into multiple files later if needed)